locust/runners.py

```diff
@@ -49,9 +51,9 @@
     StatsError,
     setup_distributed_stats_event_listeners,
 )
-from . import argument_parser
 
 if TYPE_CHECKING:
+    from . import User
```

Comment:
did this have to be moved in?

Reply:
Yes (or it depends), see this [comment](https://github.com/locustio/locust/pull/2556#issuecomment-1902687243).